src/EnergyPlus/SetPointManager.hh

```diff
@@ -185,81 +154,44 @@ namespace SetPointManager {
-        case SetPointManagerType::SZMaxHum:
-            return "SetpointManager:SingleZone:Humidity:Maximum";
-        case SetPointManagerType::MixedAir:
-            return "SetpointManager:MixedAir";
-        case SetPointManagerType::OutsideAirPretreat:
-            return "SetpointManager:OutdoorAirPretreat";
-        case SetPointManagerType::Warmest:
-            return "SetpointManager:Warmest";
-        case SetPointManagerType::Coldest:
-            return "SetpointManager:Coldest";
-        case SetPointManagerType::WarmestTempFlow:
-            return "SetpointManager:WarmestTemperatureFlow";
-        case SetPointManagerType::RAB:
-            return "SetpointManager:ReturnAirBypassFlow";
-        case SetPointManagerType::MZCoolingAverage:
-            return "SetpointManager:MultiZone:Cooling:Average";
-        case SetPointManagerType::MZHeatingAverage:
-            return "SetpointManager:MultiZone:Heating:Average";
-        case SetPointManagerType::MZMinHumAverage:
-            return "SetpointManager:MultiZone:MinimumHumidity:Average";
-        case SetPointManagerType::MZMaxHumAverage:
-            return "SetpointManager:MultiZone:MaximumHumidity:Average";
-        case SetPointManagerType::MZMinHum:
-            return "SetpointManager:MultiZone:Humidity:Minimum";
-        case SetPointManagerType::MZMaxHum:
-            return "SetpointManager:MultiZone:Humidity:Maximum";
-        case SetPointManagerType::FollowOATemp:
-            return "SetpointManager:FollowOutdoorAirTemperature";
-        case SetPointManagerType::FollowSysNodeTemp:
-            return "SetpointManager:FollowSystemNodeTemperature";
-        case SetPointManagerType::GroundTemp:
-            return "SetpointManager:FollowGroundTemperature";
-        case SetPointManagerType::CondEntReset:
-            return "SetpointManager:CondenserEnteringReset";
-        case SetPointManagerType::IdealCondEntReset:
-            return "SetpointManager:CondenserEnteringReset:Ideal";
-        case SetPointManagerType::SZOneStageCooling:
-            return "SetpointManager:SingleZone:OneStageCooling";
-        case SetPointManagerType::SZOneStageHeating:
-            return "SetpointManager:SingleZone:OneStageHeating";
-        case SetPointManagerType::ReturnWaterResetChW:
-            return "SetpointManager:ReturnTemperature:ChilledWater";
-        case SetPointManagerType::ReturnWaterResetHW:
-            return "SetpointManager:ReturnTemperature:HotWater";
-        case SetPointManagerType::TESScheduled:
-            return "SetpointManager:ScheduledTES";
-        case SetPointManagerType::Invalid:
-            return "*UNKNOWN*";
-        default:
-            assert(false);
-        }
-        return "*UNKNOWN*"; // not sure how we would get here, the switch block cases are exhaustive
-    }
+    constexpr std::array<std::string_view, static_cast<int>(SetPointManagerType::Num)> managerTypeName = {
```

Comment:
Please move these to the `.cc` file and if necessary add an `extern const` declaration here.